src/style.css

```diff
@@ -0,0 +1,25 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+h1,
+h2,
+p {
+  margin: 0;
+}
+
+.page-tittle {
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.article-tittle {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+.article-tittle .link-article {
+  color: green;
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element